oscm-app-approval/src/main/java/org/oscm/app/approval/controller/ApprovalInstanceAccess.java

```diff
@@ -20,17 +21,45 @@
 import javax.annotation.PostConstruct;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
+import java.util.Map;
+import java.util.function.Predicate;
 
 public class ApprovalInstanceAccess implements InstanceAccess {
 
-  private APPlatformService platformService;
+  private static final long serialVersionUID = -7079269432576472393L;
+  protected APPlatformService platformService;
 
   @PostConstruct
   public void initialize() {
     platformService = APPlatformServiceFactory.getInstance();
   }
 
+  public ClientData getCustomerSettings(String clientOrganizationId) throws APPlatformException {
+    ClientData[] data = new ClientData[1];
+    data[0] = new ClientData(clientOrganizationId);
+
+    Predicate<Map<String, Setting>> filter =
+        new Predicate<Map<String, Setting>>() {
+
```

Comment:
Lambda expression could be used here instead of anonymous predicate

Reply:
Done.